src/main/java/org/jenkinsci/plugins/mib3flash/Mib3Flash.java

```diff
@@ -0,0 +1,122 @@
+package org.jenkinsci.plugins.mib3flash;
+
+import java.io.IOException;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.io.InputStream;
+
+import java.util.Map;
+import java.util.TreeMap;
+
+import hudson.Extension;
+import hudson.model.ParameterDefinition;
+import hudson.model.ParameterValue;
+import hudson.model.StringParameterValue;
+import net.sf.json.JSONObject;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.StaplerRequest;
+
+public class Mib3Flash extends ParameterDefinition {
+    private String login = null;
+    private String host = null;
+    private String command = null;
+
+    @DataBoundConstructor
+    public Mib3Flash(String name, String description, String login, String host, String command) {
+        super(name, description);
+
+        this.login = login;
+        this.host = host;
+        this.command = command;
+    }
+
+    public String getLogin() {
+        return login;
+    }
+
+    public String getHost() {
+        return host;
+    }
+
+    public String getCommand() {
+        return command;
+    }
+
+    @Override
+    public ParameterValue createValue(StaplerRequest req, JSONObject jo) {
+        StringParameterValue value = req.bindJSON(StringParameterValue.class, jo);
+        value.setDescription(getDescription());
+        return value;
+    }
+
+    @Override
+    public ParameterValue createValue(StaplerRequest req) {
+        return null;
+    }
+
+    public Map<String, String> getEntries() {
+
+        Map<String, String> entries = new TreeMap<>();
+
+        
+        InputStream is = null;
+        InputStreamReader isr = null;
+        BufferedReader br = null;
+        try {
+            // Build ssh command to run
+            String ssh_command = "ssh " + login + "@" + host + " \"" + command + "\"";
+            System.out.println("Mib3FlashBoard: run command: " + ssh_command);
```

Comment:
Wystrzegaj się w jakimkolwiek kodzie sysoutów. Jeśli już musisz coś wypisywać to użyj logera tutaj [przykład](https://github.com/klimas7/dir-digger-plugin/blob/1fe19c3ea7514d94137df5a1163dcbba35c18a73/src/main/java/org/jenkinsci/plugins/dirdigger/FileTreeBuilder.java#L16)